### ISSUE TITLE
[RFQ] Monitoring support

### ISSUE THREAD

**Issue by ventilooo** [mattermost/mattermost-docker#266]
Since the CE version of mattermost doesn't expose monitoring data.

We use mattermost-docker internaly and we use telegraf/grafana/influxdb for monitoring the all stack.
We extended the docker-compose of that project and add  a telegraf config file.

But our telegraf configuration is very basic.
But it would be great addition to the project to provide out-of the box monitoring.

If the maintainers are Okay,  with that proposition, I will propose a basic MVP of that monitoring stack.
My telegraf skills are limited so the help of other contributors is more than welcome.

**Comment by pichouk** [comment]
I disagree to add more containers to the `docker-compose.yml` because I want it to be the most generic possible.
**BUT** I am thinking of a way to allow users to propose some contribution examples (and a monitoring example is a good idea). It will be part of the "[web image deprecation](https://github.com/mattermost/mattermost-docker/issues/245) project", it is stale for now because I'm running out of time but I will work on this as soon as I can.

So I'll let your request open and I will ping you when things will move on, thanks for your proposal :)

**Comment by pichouk** [comment]
We will probably deprecate the use of a `docker-compose.yml` file in this repository on #366. I'll let you know.
